### PR TITLE
Docs - fixed actions with each story

### PR DESCRIPTION
**Pull request by QuintonJason**
## Description

This pass in the documentation is aimed to correct all form fields per component: input fields, toggles, and dropdowns

This PR is **NOT** intended to resolve anything related the data in the `children` or any other field that contains an `Array`, `Objects`, or `Node(s)`

 - [x] - updated all links to react components
 - [x] - updated alert story actions
 - [x] - updated datacard story actions
 - [x] - updated modal story actions
 - [x] - updated tabs story actions
 - [x] - updated type story actions
 - [x] - updated panel story actions
 - [x] - updated avatar actions
     - [ ] - separated AvatarGroup from Avatar <- check this
 - [x] - updated copytext story
 - [x] - updated form section story
 - [x] - updated iconcard story
 - [ ] - revisit input story - many fields not working
 - [ ] - revisit label story - some field not working but props do nothing within component (maybe intentional)
 - [x] - updated link story
 - [x] - updated panelcontrols story
 - [x] - updated popover story
 - [x] - updated property story
 - [x] - updated select story
 - [x] - updated textarea story
 - [ ] - updated checkbox story
     - [ ] - props being pulled from Toggle. Args not populating
 - [ ] - updated radio story
     - [ ] - props being pulled from Toggle. Args not populating
 - [ ] - updated tooltip story
     - [ ] - props being pulled from TooltipElement. Args not populating
### Screenshots

No visual diffs

### Steps for testing
This PR is to cleanup all actions with story book, so we should now be able to click on any and all components and interact with the toggles or fields

### QA Steps
(Low) - Documentation updates to standardize, clean up, and add missing props for elements and objects.

### CODE REVIEW

File: packages/sage-react/lib/AvatarGroup/AvatarGroup.story.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { AvatarGroup } from './AvatarGroup';
+
+export default {
+  title: 'Sage/AvatarGroup',
```
Comment (philschanely) [inline packages/sage-react/lib/AvatarGroup/AvatarGroup.story.jsx:5]: Would "Avatar Group" be ok here?

File: packages/sage-react/lib/Panel/Panel.story.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { boolean, select, radios } from '@storybook/addon-knobs';
+import { select, radios } from '@storybook/addon-knobs';
```
Comment (philschanely) [inline packages/sage-react/lib/Panel/Panel.story.jsx:2]: Lets look into removing these and their use. Hope is to eventually pull this addon from the repo as its currently leading to some errors.

**Review by philschanely — APPROVED** [review]
A few comments/suggestions but looking good!

**Review by joedakroub — APPROVED** [review]
Everything on the list looks terrific; tested each one. Nice work Quinton! 👍🏼

**Pull request merged**